assets/js/common/Table/Table.jsx

```diff
@@ -76,6 +76,10 @@ function Table({
 
   const [filters, setFilters] = useState([]);
   const [currentPage, setCurrentPage] = useState(1);
+  const itemsPerPageOptions = [10, 20, 50, 75, 100];
```

Comment:
I think we can have this outside the component's scope, it would work the same and the list would be declared just once